js/DateTimePicker/Display-format.md

```diff
@@ -112,7 +112,7 @@ Add the following code in your **HTML** page.
 {% endhighlight %}
 
 
-![JavaScript DateTimePicker Display format](Display-format_images/Display-format_img1.png)
```

Comment:
<img width="671" alt="image" src="https://github.com/syncfusion-content/javascript-docs/assets/44014283/2571cc37-f35d-4446-8e02-32d198233dad">
You can change the title like as,
Display date format ---  | Syncfusion

Reply:
changed